pkg/virt-api/webhooks/s390x.go

```diff
@@ -0,0 +1,63 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+/*
+ * s390x utilities are in the webhooks package because they are used both
+ * by validation and mutation webhooks.
+ */
+package webhooks
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func setDefaultS390xDisksBus(spec *v1.VirtualMachineInstanceSpec) {
+	bus := v1.DiskBusVirtio
+
+	for i := range spec.Domain.Devices.Disks {
+		disk := &spec.Domain.Devices.Disks[i].DiskDevice
+
+		if disk.Disk != nil && disk.Disk.Bus == "" {
+			disk.Disk.Bus = bus
+		}
+		if disk.CDRom != nil && disk.CDRom.Bus == "" {
+			disk.CDRom.Bus = bus
+		}
+		if disk.LUN != nil && disk.LUN.Bus == "" {
+			disk.LUN.Bus = bus
+		}
+	}
+}
+
+// Disable ACPI Feature by default on s390x, since it is not supported
+func setS390xDefaultFeatures(spec *v1.VirtualMachineInstanceSpec) {
+	_false := false
+	featureStateDisabled := v1.FeatureState{Enabled: &_false}
+	if spec.Domain.Features == nil {
+		spec.Domain.Features = &v1.Features{
+			ACPI: featureStateDisabled,
+		}
+	} else {
+		if spec.Domain.Features.ACPI.Enabled == nil {
+			spec.Domain.Features.ACPI.Enabled = &_false
```

Comment:
here, please use `pointer.BoolPtr(false)`

Reply:
k8s.io/utils/pointer is deprecated, the new one is k8s.io/utils/ptr.
But i'm not sure if `ptr.To[bool](false)` instead of `&_false` is improving the code here. It just seems to add a new dependency and makes it harder to read.

Reply:
We have the `"kubevirt.io/kubevirt/pkg/pointer"`